### PR TITLE
add to work with async await module

### PR DESCRIPTION
**Pull request by frani**
HI Shane,  here is a resumen of changes:
- created module that work with async/await
- created module that test async/await module
- fix deprecated method
- fix variables name and replace 'var' for 'let'

### CODE REVIEW

**Comment by frani** [comment]
ups I see I was using different eslint config - I will fix later.

**Review by sffc — CHANGES_REQUESTED** [review]
Thank you!

I changed my mind about backwards compatibility.  I think we should bump the major version number and have async/await as the only API.  If people can't use async/await, then they can use an older version of the library.

Can you please:

1. Undo all the unnecessary formatting changes.  Most of this PR is formatting changes, not code changes, and it makes it very difficult to review.
2. Either remove the non-async/await code paths or make them wrap the old code paths (don't duplicate code).
3. Update the README file with new documentation.

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
-  - "4"
-  - "node"
+  - '12'
```
Comment (sffc) [inline .travis.yml:3]: Please set the version to 10, which is the oldest actively supported Node.js version.  12 is too new.

Comment (frani) [reply .travis.yml]: done

**Review by sffc — COMMENTED** [review]
To be clear, if you want to keep async/await in a separate namespace alongside the old-style code, that's fine.  But, you shouldn't *duplicate* the code, as you appear to have done in this PR.

File: test/base62.test.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
+"use strict";
```
Comment (sffc) [inline test/base62.test.js:1]: Please undo the name change of this file, or make sure it is similar enough to the old file that Git sees it as a file move.  (Also for handler.test.js)

**Review by sffc — CHANGES_REQUESTED** [review]

**Comment by sffc** [comment]
I'm closing this PR due to inactivity. Please reopen if you intend to continue with this change.

**Pull request closed without merging**